reveal/run.js

```diff
@@ -36,11 +36,25 @@
       slideNumber: false,
       mouseWheel: true,
       keyboard: {
-        37: function leftArrow() { if (isRemoteMode()) Reveal.prev(); else Reveal.left(); },
-        38: function upArrow() { if (isRemoteMode()) Reveal.prev(); else Reveal.up(); },
-        39: function rightArrow() { if (isRemoteMode()) Reveal.next(); else Reveal.right(); },
-        40: function downArrow() { if (isRemoteMode()) Reveal.next(); else Reveal.down(); },
-        82: function rKey() { toggleRemoteMode(); },
+        37: function leftArrow() {
+              if (isRemoteMode()) Reveal.prev();
+              else Reveal.left();
+            },
+        38: function upArrow() {
+              if (isRemoteMode()) Reveal.prev();
+              else Reveal.up();
+            },
+        39: function rightArrow() {
+              if (isRemoteMode()) Reveal.next();
+              else Reveal.right();
+            },
+        40: function downArrow() {
+              if (isRemoteMode()) Reveal.next();
+              else Reveal.down();
+            },
+        82: function rKey() {
+              toggleRemoteMode();
+            },
```

Comment:
I'm nitpicking, but this does not look like conventional JS indentation. It should look like this:

```js
82: function rKey() {
  toggleRemoteMode();
},
```